### ISSUE TITLE
Question about MedAL

### ISSUE THREAD

**Issue by MST9998** [adgaudio/O-MedAL#2]
Hi, these days I am working on an active learning project, and I read your paper about the combination of AL and medical images, which is really interesting and the datasets you choose to experiment with are so perfect! 

Therefore, I am trying to achieve the same result in paper MedAL: Accurate and Robust Deep Active Learning for Medical Image Analysis. I think  I used the same model (Inception V3), same data (Messidor), same data-processing (data augmentation in your paper O-MedAL), same baselines(entropy-based method) and hyper parameters as yours, but I found the result is not as stable as yours, also much higher than yours. The graph shown in your paper looks really good and the performance increases as the labeled data size gets larger smoothly... I am not sure if I did something wrong or if there are some details I missed. Is that a normal phenomenon? 

Here's my setting Adam optimizer with lr 0.0002, weight decay=0; batchsize=16; stop the retraining until the training accuracy=1... Is there anything wrong with my settings? 

THANKS!
